### ISSUE TITLE
Combining Classification and Regression with tflearn

### ISSUE THREAD

**Issue by bhavikajalli** [tflearn/tflearn#956]
As the topic says, I want to combine a classification and a regression problem.
I want to have 9 output neurons. 7 of them from the softmax activation and two of them from the linear activation. I also want to give them a different loss function.  How can I go about doing that?

```
net = tflearn.global_avg_pool(net)
# Regression
net = tflearn.fully_connected(net, 9,activation='softmax')

```
The following obviously did not work as I was unable to add the outputs from the two parts of the network to a single regression layer.

```
net1 = tflearn.fully_connected(net, 7,activation='softmax')
net = tflearn.regression(net, optimizer=opt,
                         loss='categorical_crossentropy')
net2 = tflearn.fully_connected(net, 2,activation='linear')
net = tflearn.regression(net, optimizer=opt,
                         loss='mean_square')
net = net1+net2
```
Any ideas/help is appreciated.
